### PR TITLE
PR cleanup

### PR DESCRIPTION
**Pull request by aoli-al**

### CODE REVIEW

File: Src/PChecker/CheckerCore/Coverage/CoverageInfo.cs [hunk 1]
```diff
@@ -54,8 +54,6 @@ public CoverageInfo()
             Machines = new HashSet<string>();
             MachinesToStates = new Dictionary<string, HashSet<string>>();
             RegisteredEvents = new Dictionary<string, HashSet<string>>();
-            EventInfo = new EventCoverage();
-            CoverageGraph = new Graph();
```
Comment (ankushdesai) [inline Src/PChecker/CheckerCore/Coverage/CoverageInfo.cs:58]: why this change?

File: Src/PChecker/CheckerCore/Coverage/ActorRuntimeLogEventCoverage.cs
Comment (ankushdesai) [inline Src/PChecker/CheckerCore/Coverage/ActorRuntimeLogEventCoverage.cs:1]: Can we remove this renaming?

**Review by ankushdesai — CHANGES_REQUESTED** [review]

File: Src/PChecker/CheckerCore/Coverage/CoverageInfo.cs (hunk 1 above)
Comment (aoli-al) [reply Src/PChecker/CheckerCore/Coverage/CoverageInfo.cs:58]: This change was introduced because of merge. Removing this aligns with the upstream https://github.com/p-org/P/blob/master/Src/PChecker/CheckerCore/Coverage/CoverageInfo.cs#L56

File: Src/PChecker/CheckerCore/Coverage/ActorRuntimeLogEventCoverage.cs
Comment (aoli-al) [reply Src/PChecker/CheckerCore/Coverage/ActorRuntimeLogEventCoverage.cs:1]: Yes, this is only for `p-org:experimental/feedback-strategy`, if you compare this branch with master branch you won't see this change.

**Comment by aoli-al** [comment]
I noticed that you have already merged part of this PR through #719 . Those commits are removed.
